geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -31,9 +31,14 @@ Changes to apply
    using Sass in place of Less. The major change is that the variables will be prefixed by '$'
    instead of '@' and we will use rem as recommended unit.
 
-3. Docker version: To be able to use the mapfile faster we don't use the mako anymore for that,
-   to convert them use:
+3. Docker version: To be able to deploy mapfile modifications faster we don't use the mako anymore for that.
```

Comment:
Suggestion:

Docker version: To be able to deploy mapfile modifications faster, we have switched from mako templating to a new templating mechanism.